os/kernel/clock/clock_initialize.c

```diff
@@ -339,4 +339,11 @@ void clock_timer(void)
 
 	g_system_timer++;
 }
+
+#ifdef CONFIG_SCHED_TICKSUPPRESS
+void clock_timer_nohz(int ticks)
+{
+	g_system_timer += ticks;
```

Comment:
g_system_timer is a u64 variable. But the input ticks is set as int type. What is the strategy to handle overflow cases? Even if the input type is int, shouldn't it be casted to (u64)?

Reply:
Thank you for view. Changed